.github/workflows/sign.yml

```diff
@@ -34,162 +26,38 @@ jobs:
-            {
-              "SourceLocationType": "UNC",
-              "SourceRootDirectory": "@@workspace@@\\unsigned\\beta",
-              "DestinationLocationType": "UNC",
-              "DestinationRootDirectory": "@@workspace@@\\signed\\beta",
-              "SignRequestFiles": [
-                {
-                  "SourceLocation": "${{inputs.assembly_name}}.${{inputs.beta_version}}.nupkg",
-                  "DestinationLocation": "${{inputs.assembly_name}}.${{inputs.beta_version}}.nupkg"
-                }
-              ],
-              "SigningInfo": {
-                "Operations": [
-                  {
-                    "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetSign",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  },
-                  {
-                  "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetVerify",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  }
-                ]
-              }
-            }
-          ]
-        }
-        '@
-        $workspace = [regex]::escape("${{github.workspace}}")
-        $betaInputJson = $betaInputJsonTemplate -replace '@@workspace@@', $workspace
-        Out-File -FilePath .\betaInput.json -InputObject $betaInputJson
-
-    - name: Create Stable Input File
-      run: |
-        $stableInputJsonTemplate = @'
-        {
-          "Version": "1.0.0",
-          "SignBatches": [
-            {
-              "SourceLocationType": "UNC",
-              "SourceRootDirectory": "@@workspace@@\\unsigned\\stable",
-              "DestinationLocationType": "UNC",
-              "DestinationRootDirectory": "@@workspace@@\\signed\\stable",
-              "SignRequestFiles": [
-                {
-                  "SourceLocation": "${{inputs.assembly_name}}.${{inputs.stable_version}}.nupkg",
-                  "DestinationLocation": "${{inputs.assembly_name}}.${{inputs.stable_version}}.nupkg"
-                }
-              ],
-              "SigningInfo": {
-                "Operations": [
-                  {
-                    "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetSign",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  },
-                  {
-                  "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetVerify",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  }
-                ]
-              }
-            }
-          ]
-        }
-        '@
-        $workspace = [regex]::escape("${{github.workspace}}")
-        $stableInputJson = $stableInputJsonTemplate -replace '@@workspace@@', $workspace
-        Out-File -FilePath .\stableInput.json -InputObject $stableInputJson
+    - name: Setup NuGet
+      uses: NuGet/setup-nuget@v1.0.5
 
     - name: Azure Login
```

Comment:
Do we still need the login and logout steps or the sign script is handling that?

Reply:
Yes, the signing script is only doing the login/logout when the user_login switch is provided for local testing. The original login is still useful because it takes the creds secret.